### ISSUE TITLE
Version 1.00 on gd

### ISSUE THREAD

**Issue by Pharaoh2k** [CyB1K/PS4-Fake-PKG-Tools-3.87#6]
While gengp4 is generating the XML it downgrades/changes the SFO of a GD to version 1.00 regardless of the version it had before, so if a pkg is a combined remaster of base+patch v02.13, gengp4 makes it v01.00.
Can you please fix this?

**Comment by CyB1K** [comment]
For the gp4 you always need to name the folder accordingly. If its a remaster it has to be editted to be a base 1.00, building a remaster is doable but its a pain and a mess and doesnt bring any benefits so we always build a remaster as 1.00

**Comment by Pharaoh2k** [comment]
What does it mean "building a remaster is a pain and a mess and doesnt bring any benefits so we always build a remaster as 1.00"?
I don't get why a remaster must always be 1.00? And why they're are no benefits for remasters? Instead of downloading a 40gb game + 40gb patch you just download, store and install a significantly smaller remaster of the two combined. I think that's a huge benefit for many people.

**Comment by CyB1K** [comment]
You need a base still to make a remaster pkg (even if a fake one), you cant make remaster patches, it doesnt exist and so cant link it to any "base" remaster, you would need to make a remaster pkg everytime theres a new update. Making a remaster pkg involves a lot of manual editting the gp4 otherwise its invalid on the project and like i said theres no benefit to it other than having a game sayings its version x, which you are better off just making a sce_sys and param patch with the actual version if it bothers you that much.

**Comment by Pharaoh2k** [comment]
Trainers need to match the correct game version (and CUSA/region), if a trainer is made for Ghost of Tsushima version 02.13, then it won't match and won't work on the game (which is a remaster with version 1.00) without a version fix.
This can be a critical issue, not just for me, but for an entire community.
The version field is there for a reason, and should not be set to 1.00, unless it really is 1.00
Besides, this issue only happens when using the gui tool. The gp4 library doesn't have this issue.

**Comment by CyB1K** [comment]
A trainer is usually based on the ps4 memory, as far as i know it has no relation with the version displayed on ps4, specially cuz the version it shows doesnt affect on anything, the game will be the same memory if its 1.00 or 99.10. No one builds remaster pkgs other than HK on apps but thats because its easier to merge them a remaster than include base and update as they are usually just a few mb. With that being said as long as you know which version is installed it shouldnt matter and if the trainer is really searching for version checks then it shouldnt

**Comment by Pharaoh2k** [comment]
That is incorrect. GoldHen built-in trainer (for example) relies on the game version to load, if it doesn't match - it won't attach.
Trainers must check for version number, as different version has different offsets and if a trainer is attached to the wrong version - the game will crash.
The problem is that many existing remasters (and I bet that future remasters too) will come without an additional version fix patch. This is a problem that can be avoided if gp4 doesn't force version 1.00 (again the gp4 lib doesn't do that).

**Comment by CyB1K** [comment]
Gp4 is not adapted to remasters, neither is the tools itself. Remasters are a mess and no one of us has the source of gengp4 anyway. Well idk how to solve that for you sorry. I wont be making remasters thats for sure, its too much editng for no gains and i have 0 interest in cheats
